resources/shortcodes.php

```diff
@@ -1,10 +1,9 @@
 <?php
 
-use \ZypeMedia\Controllers\Consumer;
 use Themosis\Facades\Input;
-use Themosis\Facades\Asset;
+use ZypeMedia\Controllers\Consumer;
 
-add_shortcode('zype_grid', function() {
+add_shortcode('zype_grid', function () {
     $video = new Consumer\Videos();
```

Comment:
@Osoro we should validate `Input::get('zype_type')` in [line 8](https://github.com/zyhttps://github.com/zype/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split&w=1#diff-0a60c74cf55d0fa5deac99c808642918R8e/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split&w=1#diff-0a60c74cf55d0fa5deac99c808642918R8), right?